genomedata/load_genomedata.py

```diff
@@ -238,6 +242,19 @@ def parse_cmdline(cmdline):
                             const="sizes",
                             help="sequence files contain list of sizes instead of"
                             " sequence")
+
+    chromsome_names = parser.add_argument_group("Chromosome naming")
```

Comment:
Put these in an `ArgumentParser` in a util library and use the `ArgumentParser(parents=)` option so that we don't have to repeat. See https://docs.python.org/3/library/argparse.html#parents